frontend/src/features/idir/search/table/CompanySearchResultColumnDef.tsx

```diff
@@ -47,12 +44,20 @@ export const CompanySearchResultColumnDef: MRT_ColumnDef<CompanyProfile>[] = [
         return state?.code === mailingAddress?.provinceCode
       })
 
+      const addressLine1 = getDefaultRequiredVal(null, mailingAddress?.addressLine1);
+      const addressLine2 = getDefaultRequiredVal(null, mailingAddress?.addressLine2);
+      const countryName = getDefaultRequiredVal(null, country[0]?.name);
+      const provinceName = getDefaultRequiredVal(null, province[0]?.name);
+      const cityName = getDefaultRequiredVal(null, mailingAddress?.city);
+      const postalCodeName = getDefaultRequiredVal(null, mailingAddress?.postalCode);
+
       return (
         <>
-          {mailingAddress?.addressLine1}<br />
-          {country[0]?.name}<br />
-          {province[0]?.name}<br />
-          {mailingAddress?.city} {mailingAddress?.postalCode}
+          {addressLine1 ? addressLine1 : null} {addressLine1 && <br />}
```

Comment:
Unnecessary condition, I think.

Reply:
which one? lol - had to break this up into multiple ones

Reply:
oh I think I see what you mean - thanks :bow:

Reply:
yeah, should have elaborated but yeah, getRequiredVal already handles what to set as value for the field. hence we didn't need this condition is what I meant.